src/delta5node/delta5node.ino

```diff
@@ -31,9 +31,16 @@
 // Node 5 = 16, Node 6 = 18, Node 7 = 20, Node 8 = 22
 #define i2cSlaveAddress 8
 
-const int slaveSelectPin = 10; // Setup data pins for rx5808 comms
-const int spiDataPin = 11;
-const int spiClockPin = 13;
+//#define LEONARDO
```

Comment:
maybe a comment here saying "uncomment the line for use with..."

Reply:
ok